src/origin_sdk/service/Project.py

```diff
@@ -1,14 +1,14 @@
 import logging
 from origin_sdk.OriginSession import OriginSession
 from typing import Optional
-from tempfile import TemporaryDirectory
-import pandas as pd
-from io import StringIO
 
 logger = logging.getLogger(__name__)
 
 
 class Project:
+    """The Project class is a more user-friendly set of mappings around the
```

Comment:
Are these comments for us or end users? I think this is more detail than end users would want.

Reply:
Both